ci/cron/src/Main.hs

```diff
@@ -243,5 +246,11 @@ main = do
                 Exit.exitSuccess
             else do
                 push_to_s3 docs_folder
-                let latest = Foldable.maximumBy compare_versions gh_resp
-                tell_hubspot latest
+                let gh_latest = Foldable.maximumBy (compare `Function.on` to_v . name) gh_resp
+                    docs_latest = Foldable.maximumBy (compare `Function.on` to_v) docs_resp
```

Comment:
Same question for `docs_resp`.

Reply:
Same answer: the script should crash loudly if that's the case.